integration/test_flow_integration.py

```diff
@@ -117,16 +119,67 @@ def _get_redis_kv_all_attrs(setup_teardown_test: ContextForTests, key: str):
     }
 
 
-def get_key_all_attrs_test_helper(setup_teardown_test: ContextForTests, key: str):
+def _get_sql_by_key_all_attrs(
+    setup_teardown_test: ContextForTests, key: Union[str, List[str]], key_name: Union[str, List[str]]
+):
+    import sqlalchemy as db
+
+    where_statement = ""
+    if isinstance(key, str):
+        key = key.split(".")
+    if isinstance(key, list):
+        for i in range(len(key_name)):
+            if i != 0:
+                where_statement += " and "
+            where_statement += f'{key_name[i]}="{key[i]}"'
+
+    table_name = setup_teardown_test._sql_table_name
+    engine = db.create_engine(setup_teardown_test.sql_db_path)
+    metadata = db.MetaData()
+    sql_table = db.Table(
+        table_name,
+        metadata,
+        autoload=True,
+        autoload_with=engine,
+    )
+    my_query = f"SELECT * FROM {sql_table} where ({where_statement})"
+    with engine.connect() as conn:
+        results = conn.execute(my_query).fetchall()
+    return [r._asdict() for r in results][0]
```

Comment:
Isn't this the same as
```suggestion
    return results[0]._asdict()
``
?